Documentation/ApiOverview/Hooks/EventDispatcher/Index.rst

```diff
@@ -139,7 +139,6 @@ If an extension author wants to provide a custom Event Listener, an according en
        tags:
          - name: event.listener
            identifier: 'myListener'
-           event: TYPO3\CMS\Core\Mail\Event\AfterMailerInitializationEvent
```

Comment:
please leave that in - as the rst file states: "in the vast majority of cases" this can be removed, however there may be cases where it is still required.